Sources/Session.php

```diff
@@ -34,6 +34,9 @@ function loadSession()
 	@ini_set('session.use_trans_sid', false);
 	@ini_set('arg_separator.output', '&amp;');
 
+	// Allows mods to change/add PHP settings
+	call_integration_hook('integrate_ini_settings');
```

Comment:
Having a hook at this point in `loadSession()` makes sense, if the hooked code relates to loading sessions. 

This is not, however, a good place for a generic hook to change arbitrary ini settings, so the name of this hook should not suggest that to mod developers. Please change it to `integrate_load_session`. I know it is only cosmetic, but we want to communicate the right idea with the name.

Reply:
Ok, done. Currently, it is the only way to change such variables (before starting a session). If you try to change them later, there will be a warning:
`: ini_set(): A session is active. You cannot change the session module's ini settings at this time in...`